CHANGELOG.md

```diff
@@ -14,23 +14,25 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Example section entry format
 
-- **Headline** via [#PR_NUMBER](https://github.com/apollographql/router/pull/PR_NUMBER)
+- **Headline** ([PR #PR_NUMBER](https://github.com/apollographql/router/pull/PR_NUMBER))
```

Comment:
I'd rather have us make it issue number when applicable. if anything it allows us to prepare a changelog before we open a PR, and avoir a whole CI run + getting yelled at by the changelog bot 😁

Reply:
Strongly agree

Reply:
oh i also strongly agree.  let's take that for a team convo.

Reply:
i was just making the template accurate for what we've been using. 😉